PATHFIND/UpdateNavmeshBlockingObject.md

```diff
@@ -5,18 +5,17 @@ ns: PATHFIND
 
 ```c
 // 0x109E99373F290687 0x4E9776D0
-void UPDATE_NAVMESH_BLOCKING_OBJECT(Any p0, float p1, float p2, float p3, float p4, float p5, float p6, float p7, Any p8);
+void UPDATE_NAVMESH_BLOCKING_OBJECT(int objectId, float posX, float posY, float posZ, float scaleX, float scaleY, float scaleZ, float heading, int flags);
```

Comment:
`int objectId` should be `Object object`, see [`codegen_types.lua`](https://github.com/citizenfx/fivem/blob/master/ext/natives/codegen_types.lua#L52C7-L52C13).